src/server/pfs/server/testing/block_api_server_test.go

```diff
@@ -0,0 +1,143 @@
+package testing
+
+import (
+	"fmt"
+	"math/rand"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/pachyderm/pachyderm/src/client/pkg/require"
+	tu "github.com/pachyderm/pachyderm/src/server/pkg/testutil"
+	"github.com/pachyderm/pachyderm/src/server/pkg/workload"
+)
+
+func TestPutGet(t *testing.T) {
+	t.Parallel()
+	err := tu.WithRealEnv(func(env *tu.RealEnv) error {
```

Comment:
All the tests in this file are pretty much identical to the original `pfs/server` tests, except for the two lines at the beginning and three lines at the end:
```go
t.Parallel()
err := tu.WithRealEnv(func(env *tu.RealEnv) error {
	...
	return nil
})
require.NoError(t, err)
```

The existing code for generating clients was also removed because we can now use `env.PachClient` instead.  This all applies to `pfs/server/testing/server_test.go` as well.